.github/actions/frontend/runtime/magic_url/action.yml

```diff
@@ -14,30 +14,19 @@ inputs:
     type: string
     default: '/'
 
-# All booleans in a composite action must leverage fromJSON to ensure the value are treated
-# as booleans and not strings. See issue here: https://github.com/actions/runner/issues/1483
-
 runs:
   using: "composite"
   steps:
     - name: magic_url_inputs
-      if: ${{ fromJSON(env.TEST) }}
       shell: bash
       run: |
         echo "github_token=${{ inputs.github_token }}"
         echo "migration=${{ inputs.migration_number }}"
         echo "magic_url_route=${{ inputs.magic_url_route }}"
-    - name: setup_env
-      if: ${{ !fromJSON(env.TEST) }}
-      uses: ./.github/actions/frontend/setup_env
-      with:
-        use_dev: true
     - name: npm_build
```

Comment:
Should this call the build action?

Reply:
I'm actually already [doing this](https://github.com/JupiterOne/.github/blob/7f3ad83f113826f8702288a4d5083f94873b56b4/.github/workflows/frontend_runtime_application_pr.yml#L136) in the workflow, so this is unnecessary. Good catch!